src/test/java/org/rulez/demokracia/pdengine/VoteCastUpdatableTest.java

```diff
@@ -82,4 +66,14 @@ public void castVote_does_not_record_the_proxy_id_when_updatable_is_false_and_it
 		
 		assertTrue(vote.votesCast.get(8).proxyId == null);
 	}
+	
+	private void init(boolean canUpdate) {
```

Comment:
Please choose names which tell us what the function does.

Reply:
As this init is in beginning of every test, it can be part of the setUp, and you are relieved of the terror of naming things ;)